modules/nlu/src/backend/api.ts

```diff
@@ -49,10 +34,11 @@ export default async (bp: typeof sdk, state: NLUState) => {
 
   router.post('/cross-validation/:lang', async (req, res) => {
     const { botId, lang } = req.params
-    const ghost = bp.ghost.forBot(botId)
 
-    const intentDefs = await getIntents(ghost)
-    const entityDefs = await state.nluByBot[botId].entityService.getCustomEntities()
+    const axiosForBot = axios.create(await bp.http.getAxiosConfigForBot(botId))
```

Comment:
Missing the localUrl parameter, will not work if your bot is on a different port than the external url